iotilecore/iotile/core/hw/hwmanager.py

```diff
@@ -167,13 +168,15 @@ def get(self, address, basic=False, force=None, uuid=None):
             return tile
 
         name = tile.tile_name()
-        _version = tile.tile_version()
+        version = tile.tile_version()
```

Comment:
I can't be 100% certain, since I'm not familiar with every single tile. I assume they all, even emulated ones, know that RPC as the status rpc, and will return that information... I can add a check I guess. I'm not too sure what to do if it doesn't return a version. Maybe it will just select "best match" somehow? Maybe the latest proxy?